README.md

```diff
@@ -1595,6 +1597,190 @@ for domain in domains:
     print(domain)
 ```
 
+### Create StaticBrcodes
+
+StaticBrcodes store account information via a code or an image that represents a PixKey and a few extra fixed parameters, such as an amount and a reconciliation ID. They can easily be used to receive Pix transactions.
```

Comment:
StaticBrcodes store account information via a QR code or an image that represents a PixKey and a few extra fixed parameters, such as an amount and a reconciliation ID. They can easily be used to receive Pix transactions.